raiden_contracts/contracts/MonitoringService.sol

```diff
@@ -90,18 +90,27 @@ contract MonitoringService is Utils {
 
     /// @notice Deposit tokens used to reward MSs
     /// Can be called by anyone several times and on behalf of other accounts
-    /// @param beneficiary The account benefitting from the deposit
-    /// @param amount The amount of tokens to be depositted
-    function deposit(address beneficiary, uint amount) public {
-        require(amount > 0);
+    /// @param beneficiary The account benefiting from the deposit
+    /// @param total_deposit Idempotent function that sets the total_deposit of
```

Comment:
This should explain what the parameter is, not what the function does. The `@notice` is meant for explaining what the function does.